docs/en/contribution/coding-guide.md

```diff
@@ -0,0 +1,101 @@
+# Coding guide
+
+1. Create entity classes using annotations in the `lombok` plugin (`@Data` `@Getter` `@Setter` `@NonNull` etc...) to reduce the amount of code
+
+2. If you need to use log4j to print logs in a class, preferably use the annotation `@Slf4j` in the `lombok` plugin
+
+3. Issue, pr submission specification:
+
+   > Title specification: [purpose] [module name] Description
+
+   1. pr purpose includes: `Hotfix`, `Feature`, `Improve`, `Bug`, `Docs`, `WIP`
+   2. issue purpose includes: `Feature`, `Bug`, `Docs`, `WIP`, `Discuss`
```

Comment:
`WIP`: I prefer to use github's `Draft PR`

Reply:
Good advice